docs/components/action-bar.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebarDepth: 0
+---
+
+# ActionBar
+
+The `<ActionBar>` component is a NativeScript abstraction for the Android ActionBar, iOS NavigationBar and Web Toolbar.
```

Comment:
there should be a live demo section which lets users to modify all attributes of the component
please review all other .md files in this MR